lizard_downloader.py

```diff
@@ -216,21 +217,24 @@ def show_data(self):
         """ Show the data as a new layer on the map """
 
         # Setup
-        WGS84 = "EPSG:4326"
+        wgs84 = "EPSG:4326"
         asset_types = ["pumpstations"]
         geometry_type = "Point"
         payload = {"page_size": 100}
 
         # Get the JSON containing the data from the Lizard API
         base_url = "https://demo.lizard.net/api/v2/"
-        url = "{}{}/".format(base_url, asset_types[0])
-        r = requests.get(url, params=payload).json()
+        url = "{}{}/?{}={}".format(
```

Comment:
Use urllib.urlencode for the payload:
https://docs.python.org/2/howto/urllib2.html